build.gradle

```diff
@@ -31,6 +31,7 @@ flyway {
 
 /* Migrate DB when stage task runs. */
 task stage() {
+    dependsOn flywayClean
```

Comment:
This is a good change for now.  We should create a ticket in the backlog to remove this line after development has stabilized so that every deploy doesn't erase all our data.